service/src/test/java/uk/gov/hmcts/reform/fpl/model/DirectionsTest.java

```diff
@@ -31,21 +33,28 @@
 
     @Test
     void shouldGetListOfDirectionsWithPopulatedCCDFieldsFromIndividualDirectionFields() {
-        CaseData caseData = CaseData.builder()
-            .directionsForCaseManagementOrder(Directions.builder()
-                .allPartiesCustomCMO(wrapElements(Direction.builder().build()))
-                .localAuthorityDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .respondentDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .cafcassDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .otherPartiesDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .courtDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .build())
+        Directions directions = Directions.builder()
+            .allPartiesCustomCMO(wrapElements(Direction.builder().build()))
+            .localAuthorityDirectionsCustomCMO(wrapElements(Direction.builder().build()))
+            .respondentDirectionsCustomCMO(wrapElements(Direction.builder().build()))
+            .cafcassDirectionsCustomCMO(wrapElements(Direction.builder().build()))
+            .otherPartiesDirectionsCustomCMO(wrapElements(Direction.builder().build()))
+            .courtDirectionsCustomCMO(wrapElements(Direction.builder().build()))
             .build();
 
         List<Element<Direction>> expected = wrapElements(getDirection(ALL_PARTIES), getDirection(LOCAL_AUTHORITY),
             getDirection(PARENTS_AND_RESPONDENTS), getDirection(CAFCASS), getDirection(OTHERS), getDirection(COURT));
 
-        assertThat(caseData.getDirectionsForCaseManagementOrder().getDirectionsList()).isEqualTo(expected);
+        assertThat(directions.getDirectionsList()).isEqualTo(expected);
+        assertTrue(directions.containsDirections());
```

Comment:
minor: but do we really need this assert? As we have it populated in `expected` and `getDirection` does not return `null`

Reply:
they're separate methods, so yeah I feel both assertions are valid